warmup/app.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package sample
+
+import (
+	"net/http"
+
+	"google.golang.org/appengine"
+	"google.golang.org/appengine/log"
+)
+
+func init() {
+	http.HandleFunc("/_ah/warmup", warmupHandler)
+}
+
+func warmupHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := appengine.NewContext(r)
+
+	// Do things using ctx.
```

Comment:
I was trying to think of what you'd want to do in a warmup request, though. Don't they all involve using a ctx? Otherwise you could just do it in init.

Will still make the comment more generic.